### PR TITLE
Refactor ssh key data source and documentation

### PR DESCRIPTION
**Pull request by Diptipowervs**
### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request

Relates OR Closes #0000

Output from acceptance testing:

```
$ make testacc TESTARGS='-run=TestAccXXX'

...
```

### CODE REVIEW

File: ibm/service/power/data_source_ibm_pi_key.go [hunk 1]
```diff
@@ -6,79 +6,65 @@ package power
...
-				Sensitive:  true,
-				Computed:   true,
-				Deprecated: "This field is deprecated, use ssh_key instead",
+			Attr_SSHKey: {
+				Computed:    true,
+				Description: "SSH RSA key.",
+				Sensitive:   true,
+				Type:        schema.TypeString,
 			},
 		},
```
Comment (ismirlia) [inline ibm/service/power/data_source_ibm_pi_key.go:47]: @michaelkad Are we in the clear to remove these deprecated fields?

Comment (ismirlia) [reply ibm/service/power/data_source_ibm_pi_key.go:47]: addressed

File: ibm/service/power/data_source_ibm_pi_keys_test.go [hunk 2]
```diff
@@ -31,6 +31,5 @@ func testAccCheckIBMPIKeysDataSourceConfig() string {
 	return fmt.Sprintf(`
 		data "ibm_pi_keys" "test" {
 			pi_cloud_instance_id = "%s"
-		}
-	`, acc.Pi_cloud_instance_id)
+		} `, acc.Pi_cloud_instance_id)
```
Comment (ismirlia) [inline ibm/service/power/data_source_ibm_pi_keys_test.go:34]: There's an extra space between "}" and "`" that needs to be removed on this line.

Comment (Diptipowervs) [reply ibm/service/power/data_source_ibm_pi_keys_test.go]: changes made

**Review by michaelkad — CHANGES_REQUESTED** [review]
@Diptipowervs could you resolve constant conflicts?

**Comment by Diptipowervs** [comment]
PR raised in main repo

**Pull request closed without merging**